external-crates/move/crates/move-package-alt/src/compatibility/legacy_parser.rs

```diff
@@ -292,18 +292,17 @@ fn parse_source_manifest(
 
             // IF we have one system package OR this package is a system package itself,
             // we disable implicit deps.
-            let implicit_deps = if has_system_package || is_system_package {
-                ImplicitDepMode::Disabled
+            let system_dependencies = if has_system_package || is_system_package {
+                Some(vec![])
             } else {
-                // Otherwise we enable implicit deps (unfortunately all of them)
-                ImplicitDepMode::Enabled(None)
+                None
```

Comment:
Curious about the implications of returning `None` here instead of a vec with the correct values. There's a weird discrepancy because we're hard coding the names here but then not hard-coding the defaults. Since we're already yoloing a little it's probably fine though